backend/src/main/java/com/benine/backend/http/EditPresetHandler.java

```diff
@@ -0,0 +1,102 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.Preset;
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.video.StreamNotAvailableException;
+
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class EditPresetHandler extends RequestHandler {
+  
+  /**
+   * Constructor for a new editPresetHandler that handles editing a preset.
+   */
+  public EditPresetHandler() {}
+ 
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
+          throws IOException, ServletException {
+    
+    String overwriteTag = request.getParameter("overwriteTag");
+    String overwritePosition = request.getParameter("overwriteposition");
+        
+    String camID = request.getParameter("camera");
+    IPCamera ipcam = (IPCamera) ServerController.getInstance().getCameraController()
+        .getCameraById(Integer.parseInt(camID));
+        
+    int presetID = Integer.parseInt(request.getParameter("presetid"));
+    Preset preset = ServerController.getInstance().getPresetController().getPresetById(presetID);
+   
```

Comment:
This is fixed. I also made the handle method a bit shorter but it is mostly code that requests certain parameters and calls the update methods.